util/settings.spec.ts

```diff
@@ -29,3 +34,73 @@ describe('settingsPath()', () => {
 			}),
 		).toEqual('/hello-nrfcloud/stack/context/someProperty'))
 })
+
+describe('getSettings()', () => {
+	it('should return the object with same scope', async () => {
+		const returnedValues = [
+			{
+				Name: `/${'STACK_NAME'}/${Scope.STACK_CONFIG}/key1`,
```

Comment:
```suggestion
				Name: `/hello-nrfcloud/stack/context/key1`,
```

Reply:
Avoid using placeholders in mock data, so it's more clear what the real format is supposed to be.